frontend/src/app/accountCreation/AccountCreationApp.tsx

```diff
@@ -0,0 +1,83 @@
+import { FunctionComponent, useEffect } from "react";
+import { ToastContainer } from "react-toastify";
+import { useDispatch, connect, useSelector } from "react-redux";
+import "react-toastify/dist/ReactToastify.css";
+import { Route, Switch, BrowserRouter as Router } from "react-router-dom";
+import { AppInsightsContext } from "@microsoft/applicationinsights-react-js";
+
+import { reactPlugin } from "../AppInsights";
+import PrimeErrorBoundary from "../PrimeErrorBoundary";
+import USAGovBanner from "../commonComponents/USAGovBanner";
+import { setInitialState } from "../store";
+import { getActivationTokenFromUrl } from "../utils/url";
+import PageNotFound from "../commonComponents/PageNotFound";
+
+import { PasswordForm } from "./PasswordForm/PasswordForm";
+import { SecurityQuestion } from "./SecurityQuestion/SecurityQuestion";
+import { MfaSelect } from "./MfaSelect/MfaSelect";
+import { MfaSms } from "./MfaSms/MfaSms";
+import { MfaComplete } from "./MfaComplete/MfaComplete";
+
+interface WrapperProps {
+  activationToken: string;
+}
+const AccountCreation404Wrapper: FunctionComponent<WrapperProps> = ({
+  activationToken,
+  children,
+}) => {
+  if (activationToken === undefined) {
+    return <>Loading...</>;
+  }
+  if (activationToken === null) {
+    return <PageNotFound />;
+  }
+  return <>{children}</>;
+};
+
+const AccountCreationApp = () => {
+  const dispatch = useDispatch();
+  const activationToken = useSelector((state: any) => state.activationToken);
```

Comment:
Can this be better typed as `useSelector<RootState, string>((state) => state.activationToken);`?